### PR TITLE
kubectl add job DESIRED column

### PR DESCRIPTION
**Pull request by adohe-zz**
@bgrant0607 I just add DESIRED column to job display as we mentioned here #21295 , would you mind reviewing this?

### CODE REVIEW

**Comment by bgrant0607** [comment]
Thanks, @AdoHe.

I believe Completions can be nil in the case that Parallelism is set:
https://github.com/kubernetes/kubernetes/blob/aee2eb3977cd7f9edab58c5e5f2226f9e6e0f30f/pkg/apis/batch/v1/defaults.go#L40

@erictune @soltysh Do you think it's more appropriate to print `<none>` in that case, or the value for Parallelism?

**Comment by AdoHe** [comment]
@bgrant0607 I just noticed this, I think I have more to do about this PR.

**Comment by soltysh** [comment]
> @erictune @soltysh Do you think it's more appropriate to print <none> in that case, or the value for Parallelism?

We use `Not set` in [describer](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/describe.go#L1019) let's be consistent here.

@AdoHe ping me when you're ready for review.

**Comment by AdoHe** [comment]
@soltysh I just update this, please review this.

**Comment by soltysh** [comment]
LGTM @bgrant0607 or @erictune  mind tagging it appropriately

**Comment by bgrant0607** [comment]
@soltysh We use `<none>` in several places in the resource printer. The string needs to not have spaces.

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -838,11 +838,22 @@ func printJob(job *extensions.Job, w io.Writer, options PrintOptions) error {
...
+			name,
+			*job.Spec.Completions,
+			job.Status.Succeeded,
+		); err != nil {
+			return err
+		}
+	} else {
+		if _, err := fmt.Fprintf(w, "%s\t%s\t%d",
+			name,
+			"Not set",
```
Comment (bgrant0607) [inline pkg/kubectl/resource_printer.go]: Sorry, but please change this to `<none>`.

**Comment by soltysh** [comment]
> @soltysh We use <none> in several places in the resource printer. The string needs to not have spaces.

OK, I'll create a followup to fix other places, to be consistent, does it make sense?

**Comment by AdoHe** [comment]
especially when display labels, we will use `<none>` as not exist. It should be better to always use `<none>` to keep consistent.

**Comment by soltysh** [comment]
@AdoHe sorry for the previous trouble.

LGTM @bgrant0607 or @erictune mind tagging it appropriately

**Pull request merged**